### DISCUSSION TITLE
Add support for arm containers

### DISCUSSION THREAD

**Discussion by guysoft** [tiangolo/meinheld-gunicorn-docker#132] [Questions] [answered]
Hey,
I want to run this on a raspberrypi, which means I need an ARM container (arm, armf, or aarch64, all work). Any chance of adding that?

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile: 

There's been support for Arm for a while, but I didn't come here to comment then. :nerd_face:
